app/Database/CourseQueries.hs

```diff
@@ -58,10 +58,10 @@ returnCourse lowerStr = runSqlite databasePath $ do
 
 -- | Queries the database for all information about @course@, constructs a JSON object
 -- representing the course and returns the appropriate JSON response.
```

Comment:
Please reword this docstring to remove the word "response", which is now a bit misleading.